### ISSUE TITLE
[Feature]: Dockerfile.cpu for aarch64

### ISSUE THREAD

**Issue by khayamgondal** [vllm-project/vllm#8259]
### 🚀 The feature, motivation and pitch

Please provide a Dockerfile.cpu for aarch64 systems. I have a GH200 and I want to run the CPU only inference 

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by mgoin** [comment]
First we need to implement a CPU backend for ARM. Currently we only have implementations for x86 and PowerPC backends

**Comment by shahizat** [comment]
Hi @mgoin , I hope you guys will release a version compatible with arm64 machines, such as the Nvidia AGX Orin Developer Kit. Looking forward!!

**Comment by khayamgondal** [comment]
Yeah especially now since there is more push towards ARM machines by nvidia

**Comment by mgoin** [comment]
To be clear you can use vLLM CUDA on ARM machines i.e. it is easy to get it working on Grace-Hopper. We just don't have a vLLM CPU backend for ARM machines.

**Comment by khayamgondal** [comment]
Yes, I am aware of that and actually use that. Some part of my research
focuses on CPU only - for that I am interested in vllm arm cpu
implementation
